models/BaseEntity.cfc

```diff
@@ -3583,8 +3783,9 @@ component accessors="true" {
 	 */
 	private boolean function canInsertAttribute( required string name ) {
 		var alias = retrieveAliasForColumn( arguments.name );
-		return variables._meta.attributes.keyExists( alias ) &&
-		variables._meta.attributes[ alias ].insert;
+		return variables._attributes.keyExists( alias ) &&
+		variables._attributes[ alias ].insert &&
+		!variables._attributes[ alias ].isParentColumn;
```

Comment:
This is fine as well, but any reason we don't pass `insert: false` and `update: false` when creating a parent column?

Reply:
Because we actually do insert _and_ update.  See the `save()` method updates